stacks/tests/pom.xml

```diff
@@ -15,24 +15,6 @@
 	<modules>
 		<module>org.jboss.tools.stacks.core.test</module>
 	</modules>
-       <build>
-                <plugins>
-                        <plugin>
-                                <groupId>org.eclipse.tycho</groupId>
-                                <artifactId>tycho-surefire-plugin</artifactId>
-                                <version>${tychoVersion}</version>
-                                <configuration>
-                                        <dependencies>
-                                                <dependency>
-                                                        <type>eclipse-plugin</type>
-                                                        <artifactId>org.eclipse.equinox.event</artifactId>
-                                                        <version>0.0.0</version>
```

Comment:
new 4.9.0.AM1 snapshot TP will include org.eclipse.equinox.event_1.4.300.v20180612-1048
old 4.80.1.Final TP had org.eclipse.equinox.event_1.4.200.v20180426-0845.

I don't know how they're different, just calling out that maybe with a new eclipse 4.9-based TP this will work. We could also move to Tycho 1.2 when building.

Reply:
I tried to run with newer TP with dependency on equinox.event. The same problem, ide is not even started -> no test for stacks are being run. Only removal of this dependency seems to be a solution, yet, I am not the author of this plugin, I do not have a clue why was this dependency there.